ios/samples/README.md

```diff
@@ -0,0 +1,37 @@
+# Filament sample iOS app
+
+This directory contains a sample app that demonstrates how to use the Filament API in an iOS application.
+
+## Prequisites
+
+iOS support for Filament is experimental and uses the Vulkan backend via MoltenVK. OpenGL is not supported on iOS yet. Because the iOS simulators do not support Metal, Filament can not be run on simulators.
+
+Before attempting to build for iOS, read [Filament's README](../../README.md). You must first cross-compile Filament for ARM64 on a macOS host before running the sample. The easiest way to do this is to run the following command at the root of Filament's source tree:
+
+```
+$ ./build.sh -p ios debug
+```
+
+This will automatically download Apple's iOS toolchain file for CMake and build Filament for iOS.
+
+Afterwards, you should find all the Filament libraries inside of `out/ios-debug/filament/lib/arm64`:
+
+```
+libbluevk.a
+libfilabridge.a
+libfilaflat.a
+libfilament.a
+libutils.a
+```
+
+The sample application expects these libraries to be present here. Likewise, when building the sample in Release mode, the linker looks for the libraries in `out/ios-release/filament/lib/arm64`. To build Filament in Release mode, replace `debug` with `release` in the above `build.sh` command.
+
+The pre-compiled `libMoltenVK.dylib` library must be present in `third_party/moltenvk/ios`. When building, the Xcode project automatically signs and includes this library inside the application bundle.
+
+## Xcode
+
+Open up the Xcode project at `ios/samples/hello-triangle/hello-triangle.xcodeproj`. Before building you will need to sign in to your Apple developer account and select an appropriate development team in the project editor.
```

Comment:
Is there a way to build it using `xcodebuild`?

Reply:
Yes, but unfortunately there’s two issues with that.

1. Users first have to open the editor anyway to select a development team for code signing.
2. `xcodebuild` can only build the app and cannot launch it on the device :(

Reply:
Ah, thanks, understood.